server/cli/launch.py

```diff
@@ -54,6 +55,8 @@
 @click.option("--obs-names", default=None, metavar="", help="Name of annotation field to use for observations.")
 @click.option("--var-names", default=None, metavar="", help="Name of annotation to use for variables.")
 @click.option("--host", default="127.0.0.1", help="Host IP address")
+@click.option("--no-port-scan", is_flag=True, default=False, show_default=True,
```

Comment:
I personally find this more confusing than just making `--port` optional.  Also, port scans have a funny connotation. :)